src/components/RelatiesKoppelingenTekstueel/RelatiesKoppelingenTekstueel.js

```diff
@@ -9,6 +9,16 @@ const getObjectFromRelation = (relation) => {
         : relation
 }
 
+/**
+ * Component that renders the RelatiesKoppelingenTekstueel component,
+ *
+ * @component
+ *
+ * @param {object} beleidsObject - Parameter containing the information of a beleid
+ * @param {Array} beleidsRelaties - Parameter containing a collection of beleidsRelaties.
+ * @param {Array} connectionProperties - Parameter containing a collection of connection properties.
+ * @param {Array} connectionPropertiesColors - Parameter containing a collection of connection property colors.
```

Comment:
type Object

Reply:
I've changed the parameter description and type to object